woocommerce-google-analytics-integration.php

```diff
@@ -142,9 +142,16 @@ public function load_plugin_textdomain() {
 		 * WooCommerce fallback notice.
 		 */
 		public function woocommerce_missing_notice() {
-			/* translators: 1 is the required component */
-			$error = sprintf( 'WooCommerce Google Analytics requires WooCommerce version %1$s or higher. You are using version %2$s', WC_GOOGLE_ANALYTICS_INTEGRATION_MIN_WC_VER, WOOCOMMERCE_VERSION );
-			echo '<div class="error"><p><strong>' . wp_kses_post( $error ) . '</strong></p></div>';
+			if ( defined( 'WOOCOMMERCE_VERSION' ) ) {
+				/* translators: 1 is the required component, 2 the Woocommerce version */
```

Comment:
I'm ok with leaving this comment, but the string can't actually be translated unless we wrap it in a `__()` function call